lib/gli/commands/help_modules/command_help_format.rb

```diff
@@ -13,38 +13,38 @@ def initialize(command,app,sorter,synopsis_formatter_class,wrapper_class=TextWra
         end
 
         def format
-          command_wrapper      = @wrapper_class.new(Terminal.instance.size[0],4 + @command.name.to_s.size + 3)
-          wrapper              = @wrapper_class.new(Terminal.instance.size[0],4)
+          @command_wrapper      = @wrapper_class.new(Terminal.instance.size[0],4 + @command.name.to_s.size + 3)
```

Comment:
Clean room is just one option, I didn't say it was the best. Actually, my PR does not use it at all.

The thing is, this warning is "invalid" because it is benign. But letting warnings spill all over the place during a require is not good for sure.

My patch suggestion was meant to be the less intrusive possible, and still, fix the warnings. There are plenty of ways to avoid such warnings, either one, would solve the issue.